records/understanding-the-tekton-results-retention-policy.adoc

```diff
@@ -0,0 +1,38 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="understanding-the-tekton-results-retention-policy"]
+= Understanding the Tekton Results retention policy
+:context: understanding-the-tekton-results-retention-policy
+
+toc::[]
+
+[role="_abstract"]
+
+The {tekton-results} Retention Policy Agent manages how long {tekton-results} and Records are retained in the database before they are pruned. You can configure the retention behavior by using the `tekton-results-config-results-retention-policy` config map in the `tekton-pipelines` namespace.
+
+The retention policy determines when and how old {tekton-results} and Records are automatically deleted from the database. You can define global defaults as well as fine-grained rules based on namespaces, labels, annotations, or run statuses.
+
+The following fields in the config map define the retention configuration:
+
+[options="header"]
+|===
+|Field |Description |Default
+
+|`runAt`
+|Specifies when the pruning job runs. The value must use a cron schedule format.
+|`"7 7 * * 7"` (every Sunday at 7:07 a.m.)
+
+|`defaultRetention`
+|Specifies the fallback retention period for Results and Records when no specific policy matches. The value can be a number (in days) or a duration string such as `30d` or `24h`.
+|`30d`
+
+|`maxRetention`
+|(Deprecated). Used for backward compatibility if `defaultRetention` is not set. This field will be removed in a future release.
```

Comment:
IMO we should specify the allowed values here even if it's deprecated. Also, we should clarify that this field behaves _the same way as `defaultRetention`_ so the config name is misleading. 

IIUC it must be a number in a string (e.g. "32" or "5"). 

@khrm does `maxRetention` support the same values as `defaultRetention`, meaning it supports unit suffixes like "32h" or "5d"?

Reply:
No. It supports only days without unit suffixes.